src/textures/CompressedArrayTexture.js

```diff
@@ -11,6 +11,20 @@ class CompressedArrayTexture extends CompressedTexture {
 		this.image.depth = depth;
 		this.wrapR = ClampToEdgeWrapping;
 
+		this.layerUpdates = new Set();
```

Comment:
Can you please make sure the new property is honored in `copy()`. For this, you need a copy method in both sub classes.

Reply:
Oh, and to make sure serialization works, `toJSON()` must also be implemented. `ObjectLoader.parseTextures()` should restore the set from the JSON.